windows/index.js

```diff
@@ -1,12 +1,10 @@
-if(require('electron-squirrel-startup')) return;
+if (require('electron-squirrel-startup')) return; // eslint-disable-line global-require
```

Comment:
This line has also problems. Would this code make more sense:
```javascript
const squirrel = require('electron-squirrel-startup');
// ....
if (squirrel) return;
```

Or the [usage page for electron-squirrel-startup](https://github.com/mongodb-js/electron-squirrel-startup#usage) for Babel/ES6 says we could also do:

```javascript
const squirrel = require('electron-squirrel-startup');
const {app} = require('electron');
// ....
if (squirrel) app.quit();
```
Either one will get rid of the `global-require` error the line has

Reply:
Any option will do. I wrote this code in a big rush back then. it's better to improve now